sdk/communication/azure-communication-chat/build.gradle

```diff
@@ -1,6 +1,6 @@
 ext.publishName = "Microsoft Azure Android Client Library Chat SDK For Communication Service"
 description = "This package contains the Android client library Chat SDK for Microsoft Azure Communication Service."
-version = "1.0.0-beta.6"
+version = "1.0.0-beta.9"
```

Comment:
this should be 1.0.0-beta.8.

Reply:
I mean, we're not releasing beta-8 from master, beta-8 will be from v2.

Reply:
I think the ACS team still considers the changes in master to be `1.0.0-beta.8`, which is why I changed this to `9`. Should we have the ACS libraries be version `8` in both branches?

Reply:
Interim plan was to do beta-8 release from master after merging this [pr](https://github.com/Azure/azure-sdk-for-android/pull/695). But since we have now all changes from master in v2 branch and also preview-5 features, we are planning to release beta.8 from v2 and not to do any release from master at all. 

The latest version in maven is beta.7, so if we use beta-9 for coming one then it leave a version gap in maven.